### PR TITLE
Add legacy varient to kind config patch

### PR DESCRIPTION
**Pull request by howardjohn**
The config patch provided only works on 1.15, but we test on older
kubernetes versions for backwards compatibility, so we need another
variant to support these.

This fixes postsubmit test failures like https://prow.istio.io/view/gcs/istio-prow/logs/integ-k8s-112-postsubmit-master/147

### CODE REVIEW

File: prow/config/trustworthy-jwt-12.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# This configs KinD to spin up a k8s cluster with trustworthy jwt (Service Account Token Volume Projection) feature.
+# This must be used for 1.14 and 1.13 k8s versions
```
Comment (pitlv2109) [inline prow/config/trustworthy-jwt-12.yaml]: You mean for k8s 1.12?

Comment (howardjohn) [reply prow/config/trustworthy-jwt-12.yaml]: oops

Comment (howardjohn) [reply prow/config/trustworthy-jwt-12.yaml]: fixed

**Pull request merged**